src/skills/SkillsBuilder.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import {
+  ActionRow,
+  Button,
+  Container,
+  FullscreenModal,
+} from '@edx/paragon';
+
+const SkillsBuilder = () => {
+  const onCloseHandle = () => {
+    window.history.back();
+  };
+
+  return (
+    <FullscreenModal
+      title="Skills Builder"
+      isOpen
+      onClose={onCloseHandle}
+      footerNode={(
+        <ActionRow>
+          <Button variant="tertiary">Go back</Button>
+          <ActionRow.Spacer />
+          <Button>Submit</Button>
```

Comment:
Will this FullScreenModal be included in the final product? Or is this just a space saver to confirm the route is working as intended?

My only worry is that we have hardcoded english here and I want to ensure that we don't miss this later. If "Skills Builder"/"Go back"/"Submit" are all strings that will be in the final product... can we just take care of that piece now?

Reply:
errr I checked the mocks, and these do seem to be placeholders/not final. You can disregard the above comment :)

Reply:
I overlooked this! I'll work on this now and get it into this PR so it doesn't fall under the radar again